src/main/java/org/apache/commons/lang3/ArrayUtils.java

```diff
@@ -8672,4 +8672,53 @@ public static void shuffle(final double[] array, final Random random) {
             swap(array, i - 1, random.nextInt(i), 1);
         }
     }
+
+    /**
+     * Gets an element from the array if the array is non-null and appropriately long, otherwise returns null
+     * @param <T> the component type of the array
+     * @param array   the array holding the desired element
+     * @param index  the index of the element in the array
+     * @return The element in the array at the index, or null if it is ill-formatted
```

Comment:
What does ill-formatted mean?
Can also return null if the array element is present and null.

I don't think this method is sufficiently useful to be worth adding